crypto/ui/ui_lib.c

```diff
@@ -857,47 +888,51 @@ int UI_set_result(UI *ui, UI_STRING *uis, const char *result)
             BIO_snprintf(number2, sizeof(number2), "%d",
                          uis->_.string_data.result_maxsize);
 
-            if (l < uis->_.string_data.result_minsize) {
+            if (len < uis->_.string_data.result_minsize) {
```

Comment:
I think the code would be more clear with a local variable pointing to ```uis->_.string_data```  which means pulling the struct for string_data out to toplevel as a named type.